modules/local/svdb/main.nf

```diff
@@ -134,4 +134,43 @@ process SVDB_MERGE_SINGLES {
         END_VERSIONS
         """
 
+}
+
+process SVDB_ANNOTATE_ARTEFACTS {
+    label "process_low"
+    tag "$group"
+
+    input:
+        tuple val(group), val(meta), file(vcf)
+        
+    output:
+        tuple val(group), val(meta), file("${meta.id}.cnv.artefacts.vcf"),  emit: artefacts
+        path "versions.yml",                                              emit: versions
+
+    when:
+        task.ext.when == null || task.ext.when
+
+    script:
+        def args = task.ext.args ?: ''
+        def prefix = task.ext.prefix ?: "${meta.id}" 
+        """
+        svdb --query --bnd_distance 25000 --overlap 0.7 --in_occ Obs --out_occ ACOUNT --in_frq Frq --out_frq AFRQ --db $params.loqusdb_export --query_vcf $vcf > ${meta.id}.cnv.artefacts.vcf
```

Comment:
Can this parameters intialized in CNV_ANNOTATE:SVDB_ANNOTATE_ARTEFACTS config ?

Reply:
yes, I was lazy. You caught me =)